more_autograding_examples/python_random_input_output/config/config.json

```diff
@@ -0,0 +1,68 @@
+{
+    "testcases" : [
+        {
+            "title" : "Simple Test Case -- No Additional Input",
+            "details" : "python *.py",
+            "command" : "python *.py",
+            "points" : 3,
+            "validation" : [
+                {
+                    "command" : ["python solution.py 1> output.txt"],
+                    "method" : "diff",
+                    "actual_file" : "STDOUT.txt",
+                    "description" : "Program Output",
+                    "expected_file" : "output.txt"
+                }
+            ]
+        },
+        {
+            "title" : "Additional Input From File -- 1 2 3 4",
+            "details" : "python *.py input.txt",
+            "command" : "python *.py input.txt",
+            "points" : 3,
+            "validation" : [
+                {
+                    "command" : ["python solution.py input.txt 1> output.txt"],
+                    "method" : "diff",
+                    "actual_file" : "STDOUT.txt",
+                    "description" : "Program Output",
+                    "expected_file" : "output.txt"
+                }
+            ]
+        },
+        {
+            "title" : "Random Input #1",
+            "command" : "python *.py randominput.txt",
+            "input_generation_commands": [
+                "python generator.py 1> randominput.txt"
```

Comment:
No , it is not pipe to output file because name of output file will be required for future application so that Instructor can use them for(output generation from student, Instructor code)